packages/react-components/react-text/package.json

```
@@ -35,7 +35,7 @@
   "dependencies": {
     "@fluentui/react-shared-contexts": "^9.4.0",
     "@fluentui/react-theme": "^9.1.8",
-    "@griffel/react": "^1.5.2",
+    "@griffel/react": "^1.5.7",
     "@fluentui/react-utilities": "^9.9.0",
     "@fluentui/react-jsx-runtime": "9.0.0-alpha.4",
```

_nit:_ Move this after the fluentui dependencies on line 40.

```suggestion
    "@fluentui/react-utilities": "^9.9.0",
    "@fluentui/react-jsx-runtime": "9.0.0-alpha.4",
    "@griffel/react": "^1.5.7",
```